test/intl402/Intl/getCanonicalLocales/returned-object-is-an-array-with-frozen-elements.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.getcanonicallocales
+description: Tests that the value returned by getCanonicalLocales is an Array
+with each element frozen that can be expanded.
+info: |
+  8.2.1 Intl.getCanonicalLocales (locales)
+  1. Let ll be ? CanonicalizeLocaleList(locales).
+  2. Return CreateArrayFromList(ll).
+includes: [compareArray.js]
+---*/
+
+var locales = ['en-US', 'fr'];
+var result = Intl.getCanonicalLocales(locales);
+var len = result.length
+
+// Individual elements should not be mutable.
+for (var key in result) {
+  var desc = Object.getOwnPropertyDescriptor(result, key);
+  assert.sameValue(desc.writable, false, 'element is not writable');
+  assert.sameValue(desc.configurable, false, 'element is not configurable');
+  assert(desc.enumerable, 'element is enumerable');
```

Comment:
Sorry that the polarity of my comment was unclear! It is a mutable array; I was talking about what form other locales are returned in.